wtxmgr/example_test.go

```diff
@@ -155,7 +155,12 @@ func Example_basicUsage() {
 	}
 
 	// Create (or open) the transaction store in the provided namespace.
-	s, err := wtxmgr.Create(ns)
+	err = wtxmgr.Create(ns)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	s, err := wtxmgr.Open(ns)
```

Comment:
wtxmgr doesn't have a Close method, so perhaps this is right.  I disagree about waddrmgr (because Close is needed), and the change to wtxmgr was only done for consistency.